### DISCUSSION TITLE
Compatibility with Last.fm

### DISCUSSION THREAD

**Discussion by metademaravilha** [fast4x/RiMusic#4935] [Ideas]
Hello everyone!

I would like to share my idea about a feature that should be added to RiMusic, and that feature is Last.fm compatibility.

Adding Last.fm compatibility to RiMusic would be simply amazing, especially for people like me who love Last.fm and like to save statistics about the songs we listen to.

You can already log in with YouTube Music, Piped, Discord and now all that's missing is Last.fm to make the app perfect.

**Comment by fast4x** [comment]
Ok thanks, maybe in the future.
